pages/travel.md

```diff
@@ -5,6 +5,10 @@ description: Information for participants traveling to Princeton
 permalink: /travel/
 ---
 
+## COVID-19 Policy
+All participants must to adhere to current Princeton University [guidelines, policies, and protocols for visitors and conferences](https://covid.princeton.edu/policies).
+As of November, 2021, this would require attesting to vaccination and wearing masks while indoors except when alone in a room or actively eating/drinking.
+
 
 ## General Notes
 The workshop will be held at the Nassau Inn in Princeton NJ located adjacent to
```

Comment:
Will the sessions be in the Nassu Inn or will they be on campus at Princeton?

Reply:
The workshop proper will be contained within the Nassau Inn. We're looking to have a dinner/reception on campus Tuesday night. If you are asking because of covid policy, because this is considered a Princeton sponsored event (sorry) we have to follow Princeton rules regardless of location. 
That said, do you think we need to add anything to this point?

Reply:
Makes sense. I was just wondering because the other event I attended at Princeton a few years back was held on campus.